packages/x-charts/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-charts",
-  "version": "6.19.5",
+  "version": "6.19.6",
```

Comment:
It doesn't look like this package received any changes since the last release. 🙈 
```suggestion
  "version": "6.19.5",
```

Reply:
But then the versions don't match and I find that more confusing.

Reply:
What about `@mui/x-tree-view` and `@mui/x-license` then? 🤔

Reply:
Restored & committed but github seems to lag behind even though the commits have been pushed. To be confirmed.